# goal definition
PillarPose pillar_pose
---
# result definition
---
# feedback